resources/skins/nudus/html_wrapper.html

```diff
@@ -0,0 +1,237 @@
+        padding: 0 .25em;
+        word-break: break-word;
+    }
+    pre {
+        --horiz: 13px;
+
+        border-radius: 3px;
+        margin 0;
+        overflow-x: auto;
+        padding: 7px var(--horiz);
+        white-space: pre !important;
+        width: calc(100wv - calc(var(--horiz) * 2)) !important;
+    }
+    pre code {
+        border: none;
+        display: block;
+        padding: 0;
+        tab-size: 4;
+    }
+    table {
+        border-collapse: collapse;
+        width: 100%;
+    }
+    table,
+    th,
+    td {
+        border: 1px solid var(--grey1);
+    }
+    img {
+        max-height: 100%;
+        /* there are few cases when in horiz layout max-width: 500px; is better with hi-res images*/
+        max-width: 100%;
+    }
+    details > summary {
+        border-radius: 5px;
+        cursor: pointer;
+    }
+    details > summary:hover {
+        background: var(--grey20);
+    }
+    details[open] > summary {
+        background: var(--grey20);
+    }
+    details > summary:focus {
+        box-shadow: none;
+        outline: none;
+    }
+    hr {
+        background: var(--grey2);
+        border: none;
+        display: block;
+        height: 2px;
+        margin: var(--base-unit) 0;
+    }
+    iframe {
+        max-width: 100%;
+        max-height: auto;
+    }
+    .rssguard-mwrapper {
+        padding: var(--base-unit) !important;
+    }
+    .rssguard-mhead .rssguard-msmall,
+    .rssguard-mhead .rssguard-mlinks {
+        opacity: .8;
+    }
+    .rssguard-mhead h1 {
+        margin: 0;
+    }
+    .rssguard-msmall {
+        font-size: 0.9em;
+    }
+    .rssguard-mlinks a {
+        word-break: break-word;
+    }
+    .rssguard-murl {
+        visibility: hidden;
+    }
+    body:hover .rssguard-mwrapper .rssguard-murl {
+        visibility: visible;
+    }
+    .rssguard-mbody {
+        /*
+         * for articles without html elements;
+         * not sure if I want to add this
+         */
+        /*word-break: break-word !important;*/
+    }
+    </style>
+
+    <style>
+        html, body, div, span, applet, object, iframe,
+        h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+        a, abbr, acronym, address, big, cite, code,
+        del, dfn, em, img, ins, kbd, q, s, samp,
+        small, strike, strong, sub, sup, tt, var,
+        b, u, i, center,
+        dl, dt, dd, ol, ul, li,
+        fieldset, form, label, legend,
+        table, caption, tbody, tfoot, thead, tr, th, td,
+        article, aside, canvas, details, embed,
+        figure, figcaption, footer, header, hgroup,
+        menu, nav, output, ruby, section, summary,
+        time, mark, audio, video {
+            font: inherit;
+            font-size: 100%;
+        }
+
+        html {
+            height: 100vh;
+        }
+
+        body {
+            height: 100vh;
+        }
+    </style>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>%1</title>
+</head>
+<body>

```

Comment:
`head`, `body` not indented etc.

Reply:
Yes, I know.

Reply:
OK, when you are done, let me know and I will merge. :) Look nice.

Reply:
> `head`, `body` not indented etc.

Do you really think they need to be?

I reviewed some other Qt projects that also use qt-webengine, and they make no indent for first-level items, but they do leave an empty lines in-between.

So should I indent them after all?

Reply:
No, of course they don't. You can even make all those files completely one-liners (to minify file size).

It is purely matter of readability, no other impacts. To me, `body` is XML/HTML element like any other, why it should not be indented?

Anyway as stated before, keep the structure whatever suits you, I will probably give you "maintainer" rights after this PR so that you can push to repo directly without PRs.

Reply:
> No, of course they don't. You can even make all those files completely one-liners [...]
> Anyway as stated before, keep the structure whatever suits you,

That is not what I mean, as I would like to keep it consistent everywhere. (If we disregard other themes dirs.)


> No, of course they don't. 

Look, for example, here https://github.com/qbittorrent/qBittorrent/blob/master/src/webui/www/private/index.html
Although they use a 4-spaces indents, perhaps because of that? since the document would not look nice with such an indent?

Reply:
As I said, indendation is completely thing of taste. Indendation of HTML has no effect on the visual outcome of rendered document.

Reply:
While what I said was "That is not what I mean, I would like to keep it consistent" (for you), and I am asking therefore.

> Indendation of HTML has no effect on the visual outcome

Of course it is not.

Reply:
Oh well, I'll keep it as it currently is.

Reply:
> I will probably give you "maintainer" rights after this PR so that you can push to repo directly without PRs.

Oh, if it goes to this I rather change the indentations level back...

Reply:
I invited you to be collaborator.

There is really nothing special to setup but you can activate Git hooks I use withing this repository, they are stored here: https://github.com/martinrotter/rssguard/tree/master/resources/scripts/hooks

Reply:
I am not really sure what I can do here, as I know no C++,  and I have no idea how to use that thing by the link.

But I am a bit tired of spamming you issue tracker, with either my issues and our conversations, and I believe you also.

Perhaps we can have some other means of communication? I don't know... Some IRC, all the talks about some libera now (never used IRC at all), e-mail... or perhaps even Discord (never really liked/used it)?

Reply:
That's not a problem, but if you just see something obvious with skin css/html etc., you can fix right away.

I had IRC channel on freenode, but it was empty for majority of day. Almost no-one used it. Perhaps I can setup some discord rom or something? Not sure what modern and easy-to-use options are there.

Reply:
> Perhaps I can setup some discord rom or something?

Well, some time ago I also needed to do it, and it was fairly easy, but... Well, I guess to start with Discord should be good enough for now, since it works from browser really well.

Reply:
> even via web UI.

Yeah, there are just too many changes recently...

Reply:
https://discord.gg/edSjECcB